.github/workflows/tests.yml

```diff
@@ -51,5 +51,10 @@ jobs:
               run: cd ../supertokens-root && ./loadModules
             - name: Setting up supertokens-root test environment
```

Comment:
Changes left:
- Integration tests
- Changelog
- Example app update and manual test with auth-react SDK (session_grant branch)